doc/dev/index.md

```diff
@@ -63,6 +63,14 @@ cd src
 # make `lean` etc. point to stage0 anywhere inside `src`
 elan override set lean4-stage0
 ```
+
+Note that `lake build` will not work on this kind of linked elan toolchain because
+it will be missing the bundled clang compiler and headers and libraries.
```

Comment:
If you built Lean from a terminal, that same compiler should be usable for driving `leanc`. I don't want to worry about setting up the bundled toolchain on developers' machines, this was strictly intended for CI.

Reply:
ok, but my Note above is true right? At least on windows it doesn't seem to work and I assume that's expected.  I can use still this local build in vscode as a lean --server though.

Reply:
You didn't tell me about your terminal or compiler, so can't really say anything about that. I assume it works out of the box in a mingw64 terminal, with `cc` in the PATH.

Reply:
Oh, I see you are right, it does work in mingw64 .  Just not outside, whereas your normal toolchain install does work outside.

Reply:
Adding clang to the PATH on windows might work, but I find these are missing from `lean-llvm-x86_64-w64-windows-gnu.tar.zst`:
- libc++.dll
- zlib1.dll

 do you know where you get those from? I don't see any mention of these in `\lean4\script\prepare-llvm-mingw.sh`
![image](https://user-images.githubusercontent.com/18707114/144690954-26ae34d3-91ed-4099-a248-bcf81fc3b9b0.png)

Reply:
Ok, I added the disclaimer here where it belongs:
https://github.com/leanprover/lean4/pull/842/commits/fc9136f24347eb040157bc7a43bcb012efbea537

Reply:
> Ok, I added the disclaimer here where it belongs

Then we can remove this paragraph?

Reply:
> do you know where you get those from?

https://github.com/leanprover/lean4/blob/171624d7a797976e79a546f754e8c4771ee52b20/script/prepare-llvm-mingw.sh#L21

Reply:
Thanks and paragraph removed.